### PR TITLE
Use GitHub actions

### LINKED ISSUE #13: Migrate from travis-ci to GitHub actions

### PR DESCRIPTION
**Pull request by kohrVid**
Closes #13 .

This PR replaces our Travis CI configuration with a Github Action workflow. The workflow has been set up to run rspec tests with the following rubies:

* 2.3
* 2.4
* 2.5
* 2.6
* 2.7

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -1 +1 @@
-service_name: travis-ci
+service_name: github-actions
```
Comment (hammady) [inline .coveralls.yml:1]: Remove if unnecessary.

**Review by hammady — CHANGES_REQUESTED** [review]

File: rayyan-scrapers.gemspec [hunk 2]
```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '~> 13'
   spec.add_development_dependency 'rspec', '~> 3.5'
   spec.add_development_dependency 'coderay', '~> 1.1'
-  spec.add_development_dependency 'coveralls', '~> 0.8'
+  spec.add_development_dependency 'simplecov', '~> 0.14.1'
```
Comment (hammady) [inline rayyan-scrapers.gemspec:26]: Why not 0.16?

Comment (kohrVid) [reply rayyan-scrapers.gemspec:26]: I just used the version that was replaced in the core repo. I can try upgrading to 0.16 if you'd prefer

Comment (hammady) [reply rayyan-scrapers.gemspec:26]: I noticed that this repo was using 0.16, that's why I was curious to know why we downgraded it.

Comment (kohrVid) [reply rayyan-scrapers.gemspec:26]: No worries - I just copied and pasted from the earlier PR. I think v0.17.1 still supports ruby 2.3 so we _could_ use that

**Comment by hammady** [comment]
@kohrVid please update the PR descriptions on all repos (ignoring coveralls being down) so that I can merge.

**Review by hammady — APPROVED** [review]

**Review by hammady — APPROVED** [review]

**Pull request merged**